src/pyodide/internal/emscriptenSetup.ts

```diff
@@ -0,0 +1,211 @@
+/**
+ * This file is intended to be executed in the Python pool (once it exists). As such, it cannot
+ * import anything that transitively uses C++ extension modules. It has to work in a vanilla v8
+ * isolate. Also, we will have to bundle this file and all of its transitive imports into a single
+ * js file.
+ */
+
+import { reportError } from 'pyodide-internal:util';
+
+/**
+ * _createPyodideModule and pyodideWasmModule together are produced by the
```

Comment:
fix comment?

Reply:
Well what it says is still true and probably even more helpful now that they are imported in separate files.